js-exercises/deep-copy-object/deepCopyObject.js

```diff
@@ -0,0 +1,12 @@
+const deepCopyObject = (objToCopy) => {
+  if (typeof (objToCopy) !== 'object' || objToCopy === null) { return objToCopy; }
+  const deepObject = {};
+  for (const key in objToCopy) {
+    if (Object.prototype.hasOwnProperty.call(objToCopy, key)) {
```

Comment:
Why not using `objToCopy.hasOwnProperty(key)`?

Reply:
@satyambnsal  if the object pass has key with name `**hasOwnProperty**`, then **objToCopy.hasOwnProperty(key)** will fail also  
**Object.prototype.hasOwnProperty.call(objToCopy, key)** is eslint suggested way to call **hasOwnProperty**